src/components/Node/Node.tsx

```diff
@@ -19,17 +19,17 @@ import {
 interface NodeProps {
   name: string;
   pathwayState: State;
-  isCurrentNode: boolean;
   xCoordinate: number;
   yCoordinate: number;
   expanded?: boolean;
   onClick?: (nodeName: string) => void;
+  currentNode: State;
 }
 
 const Node: FC<NodeProps & { ref: Ref<HTMLDivElement> }> = memo(
   forwardRef<HTMLDivElement, NodeProps>(
     (
-      { name, pathwayState, isCurrentNode, xCoordinate, yCoordinate, expanded = false, onClick },
+      { name, pathwayState, xCoordinate, yCoordinate, expanded = false, onClick, currentNode },
```

Comment:
I know this wasn't you but can we rename `name` here in these props. It is really the `nodeKey`